tests/QAN/overview_test.js

```diff
@@ -219,7 +219,6 @@ xScenario(
     ];
 
     for (const i in columnsToAdd) {
-      I.click(qanOverview.buttons.addColumn);
```

Comment:
Because we were calling this every time we called qanOverview.addSpecificColumn(). So I just moved this line inside this method thus reducig code repetition